tests/python/contrib/test_opencl/test_run_gtests.py

```diff
@@ -28,6 +29,7 @@
 # for example to run all "foo" tests twice and observe gtest output run
 # pytest -sv <this file> --gtests_args="--gtest_filter=*foo* --gtest_repeat=2"
 @tvm.testing.requires_opencl
+@pytest.mark.xfail(raises=AttributeError)
```

Comment:
Why do we need to mark this test as xfail?

Reply:
I couldn't enable opencl cpp tests due to problems with CI environment. For enabling openCL Gtests we need to get source of googletests but there are no sources in the docker image. I also tried to get source from git (you can check my some commits for this) but when we reinstall gtest lib, other CPP tests become broken. So I decided to mark this test as xfail.